### PR TITLE
Add CryptoX 1.2.0 for Android

### PR DESCRIPTION
**Pull request by Radiokot**
## Purpose

Add CryptoX 1.2.0 for Android

## Changes

- Add CryptoX 1.2.0 for Android
- Fix incorrectly rendered dropdown

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.

### CODE REVIEW

**Comment by Radiokot** [comment]
I don't know why this happens
image

File: source/mainnet/net/release-notes/release-notes-lp.rst [hunk 1]
```diff
@@ -18,36 +18,64 @@ Wallets
 
 |cryptox| for Android
 ---------------------
-    **1.1.1 - June 11, 2024**
+    **1.2.0 - August 27, 2024**
 
-    Version 1.1.1 improves WalletConnect pairing stability in situations where the connection is poor
+    This update introduces support for Company ID, CCD listings, and a newsfeed, along with optional anonymous analytics. It also fixes several visual bugs.
```
Comment (mh-concordium) [inline source/mainnet/net/release-notes/release-notes-lp.rst:23]: Should we use anonymous?

**Review by mh-concordium — COMMENTED** [review]
Just had one comment.

File: source/mainnet/net/release-notes/release-notes-lp.rst (hunk 1 above)
Comment (Radiokot) [reply source/mainnet/net/release-notes/release-notes-lp.rst:23]: I don't see an issue here, this sentence doesn't mention any crypto like it was in the dialog message

**Review by mh-concordium — APPROVED** [review]
Ok.

File: source/mainnet/net/release-notes/release-notes-lp.rst (hunk 1 above)
Comment (mh-concordium) [reply source/mainnet/net/release-notes/release-notes-lp.rst:23]: Ok.

**Pull request merged**